<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTrainingDlg</class>
 <widget class="QDialog" name="AddTrainingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add training</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/icons/floco-128.png</normaloff>:/icons/icons/floco-128.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Team:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxTeam"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxEveryWeek">
         <property name="text">
          <string>every week</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDay">
         <item>
          <property name="text">
           <string>Monday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tuesday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wednesday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thursday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Friday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Saturday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sunday</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="editDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>d.M.yyyy</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="editTime">
       <property name="time">
        <time>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Place:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editPlace"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/add-22.png</normaloff>:/icons/icons/add-22.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddAndClose">
       <property name="text">
        <string>Add and close</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/add-22.png</normaloff>:/icons/icons/add-22.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/save-22.png</normaloff>:/icons/icons/save-22.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/cancel-22.png</normaloff>:/icons/icons/cancel-22.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddTrainingDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEveryWeek</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxDay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEveryWeek</sender>
   <signal>toggled(bool)</signal>
   <receiver>editDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
